builtin/providers/aws/config.go

```diff
@@ -30,6 +31,7 @@ type AWSClient struct {
 	ec2conn         *ec2.EC2
 	elbconn         *elb.ELB
 	autoscalingconn *autoscaling.AutoScaling
+	asgconn         *awsASG.AutoScaling
```

Comment:
we need a separate connection for the meantime, until Launch Config. is converted. LC is a non-trivial size resource so I didn't bundle it hear